src/software/thunderscope/thunderscope_main.py

```diff
@@ -252,7 +252,9 @@
             {"proto_class": ObstacleList},
             {"proto_class": PathVisualization},
             {"proto_class": PassVisualization},
+            {"proto_class": AttackerVisualization},
             {"proto_class": CostVisualization},
+            {"proto_class": DebugShapesMap},
```

Comment:
DebugShapesMap doesn't seem to exist

Reply:
Oh good call, forgot that CI doesn't test this route. Updated it.